packages/amplify-category-api/src/index.ts

```diff
@@ -267,7 +267,7 @@ export async function transformCategoryStack(context: $TSContext, resource: $TSO
     if (canResourceBeTransformed(resource.resourceName)) {
       const backendDir = pathManager.getBackendDirPath();
       const overrideDir = path.join(backendDir, resource.category, resource.resourceName);
-      const isBuild = await buildOverrideDir(backendDir, overrideDir).catch(error => {
+      const isBuild = buildOverrideDir(backendDir, overrideDir).catch(error => {
```

Comment:
I think it's safer to await the promise here, no?

Reply:
I guess its the old way since we don't need both async/await and .then/catch

Reply:
The code isn't using `.then()` so I think you're sending a promise, not a boolean to the overrideFlag below. I think using full async/await (like below) and wrap in a try/catch would be better

Reply:
ill change to previous since as its preset in all overridden categories